internal/cluster/query.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/buildkite/cli/v3/internal/graphql"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+)
+
+func QueryCluster(ctx context.Context, OrganizationSlug string, ClusterID string, f *factory.Factory) (Cluster, error) {
+
+	q, err := graphql.GetClusterQueues(ctx, f.GraphQLClient, OrganizationSlug, ClusterID)
+
+	if err != nil {
+		fmt.Println("Unable to read Cluster Queues: ", err.Error())
+		return Cluster{}, err
+	}
+
+	ClusterDescription := q.Organization.Cluster.Description
+
+	cluster := Cluster{
+		OrganizationSlug: OrganizationSlug,
+		ClusterID:        ClusterID,
+		Name:             q.Organization.Cluster.Name,
+		Description:      string(*ClusterDescription),
+		Queues:           []Queue{},
+	}
+
+	for _, edge := range q.Organization.Cluster.Queues.Edges {
```

Comment:
I think an edge can be `null` if no queue are found, or it can return an empty list. Should we check that the length is `> 0`?

Reply:
let me check

Reply:
Just did a test. cluster gets created with single queue by default and even the default one is deleted code is not broken as it is a for loop which doesn't run.

<img width="793" alt="image" src="https://github.com/buildkite/cli/assets/40382124/fecdbea5-075b-4c75-9f42-808f3b0fc4c6">

Reply:
Can we output `no queues` or something if the count of queues is 0?

Reply:
Yea can do

Reply:
Done, added message when no queues are found,

<img width="763" alt="image" src="https://github.com/buildkite/cli/assets/40382124/f1a92e48-8919-40df-a3bc-1c5f7d66f13c">

Reply:
So good